ethereum/test/test_basic_inbound_channelV2.js

```diff
@@ -0,0 +1,106 @@
+const BigNumber = require('bignumber.js');
+const { ethers } = require("ethers");
+require("chai")
+  .use(require("chai-as-promised"))
+  .use(require("chai-bignumber")(BigNumber))
+  .should();
+
+const BasicInboundChannelV2 = artifacts.require("BasicInboundChannelV2");
+const MerkleProof = artifacts.require("MerkleProof");
+const ScaleCodec = artifacts.require("ScaleCodec");
+const { createBeefyValidatorFixture, runBeefyLightClientFlow } = require("./beefy-helpers");
+
+const {
+  deployBeefyLightClient
+} = require("./helpers");
+const fixture = require('./fixtures/full-flow-basicV2.json');
+
+describe("BasicInboundChannelV2", function () {
+  const interface = new ethers.utils.Interface(BasicInboundChannelV2.abi)
+
+  before(async function () {
+    const merkleProof = await MerkleProof.new();
+    const scaleCodec = await ScaleCodec.new();
+    await BasicInboundChannelV2.link(merkleProof);
+    await BasicInboundChannelV2.link(scaleCodec);
+
+    const totalNumberOfValidatorSigs = 100;
+    const beefyFixture = await createBeefyValidatorFixture(
+      totalNumberOfValidatorSigs
+    )
+    this.beefyLightClient = await deployBeefyLightClient(beefyFixture.root,
+      totalNumberOfValidatorSigs);
+
+    await runBeefyLightClientFlow(fixture, this.beefyLightClient, beefyFixture, totalNumberOfValidatorSigs, totalNumberOfValidatorSigs)
+  });
+
+  describe("submit", function () {
```

Comment:
Lets mark these tests to be skipped, since we don't have valid fixture data to make them pass. We can revisit the tests again later once the fixture data is available.

```suggestion
  describe.skip("submit", function () {
```